reference/functions/getbundlemetatags.markdown

```diff
@@ -0,0 +1,39 @@
+---
+layout: default
+title: getbundlemetatags
+published: true
+---
+
+[%CFEngine_function_prototype(bundlename, optional_tag)%]
+
+**Description:** Returns the list of [`meta`][Promise types#meta] tags for bundle `bundlename`.
+
+[%CFEngine_function_attributes(bundlename, optional_tag)%]
+
+The `optional_tag` can be used to look up a specific tag's value. If you have
+```
```

Comment:
I wonder if we need `cf3` for syntax highlighting :thinking: 
```suggestion
```cf3
```

Reply:
Looks like it's used elsewhere

Reply:
```cf3 or ```cfengine3 should work IIRC